.github/CODEOWNERS

```diff
@@ -191,14 +187,13 @@
 
 /code/__HELPERS/logging/ @dragomagol @ZephyrTFA
 
-/code/__DEFINES/MC.dm @LemonInTheDark @MrStonedOne
-/code/controllers/admin.dm @LemonInTheDark @MrStonedOne
-/code/controllers/master.dm @LemonInTheDark @MrStonedOne
-/code/controllers/failsafe.dm @LemonInTheDark @MrStonedOne
-/code/controllers/subsystem.dm @LemonInTheDark @MrStonedOne
-
-/code/controllers/subsystem/air.dm @LemonInTheDark @MrStonedOne
+/code/__DEFINES/MC.dm @LemonInTheDark
+/code/controllers/admin.dm @LemonInTheDark
+/code/controllers/master.dm @LemonInTheDark
+/code/controllers/failsafe.dm @LemonInTheDark
+/code/controllers/subsystem.dm @LemonInTheDark
```

Comment:
these should be moved back up, ideally w timer added under me.
If that's too much of a pain I can just open my own pr, dealer's choice!

Reply:
unrelated, please for the love of god stop listening to your sources (whoever they are) they're either wrong or lying to you to rile you up.

Reply:
#89192